Description: hard-coded browser and editor are replaced by 
 default installed in system.
Author: Anton Gladky <gladky.anton@gmail.com>
Bug-Debian: http://bugs.debian.org/618541
Last-Update: 2011-03-19

--- a/Common/DefaultOptions.h
+++ b/Common/DefaultOptions.h
@@ -83,7 +83,7 @@
 #elif defined(__APPLE__)
     "open -t %s" ,
 #else
-    "emacs '%s' &" ,
+    "sensible-editor '%s' &" ,
 #endif
     "System command to launch a text editor" },
   { F|S, "TmpFileName" , opt_general_tmp_filename , ".gmsh-tmp" ,
@@ -95,7 +95,7 @@
 #elif defined(__APPLE__)
     "open %s" ,
 #else
-    "firefox %s &" ,
+    "sensible-browser %s &" ,
 #endif
     "System command to launch a web browser" },
 
